rust/processor/src/db/parquet/models/default_models/parquet_block_metadata_transactions.rs

```diff
@@ -0,0 +1,151 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::{
+    bq_analytics::generic_parquet_processor::{GetTimeStamp, HasVersion, NamedTable},
+    db::common::models::default_models::raw_block_metadata_transactions::{
+        BlockMetadataTransactionConvertible, RawBlockMetadataTransaction,
+    },
+};
+use allocative_derive::Allocative;
+use field_count::FieldCount;
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+#[derive(
+    Allocative, Clone, Debug, Default, Deserialize, FieldCount, Serialize, ParquetRecordWriter,
+)]
+pub struct BlockMetadataTransaction {
```

Comment:
@ying-w

Reply:
nice to haves would be

1. end_txn_version
2. num_user_transactions

Reply:
synced offline, this is not possible without querying. so we are okay with not adding here